app/views/admin/reviews/_review_fields.erb

```diff
@@ -1,10 +1,10 @@
 <%= f.input :title %>
-<%= f.input :url %>
+<%= f.input :url, hint: 'Enter "https://print.publication" if this review only appeared in print.' %>
```

Comment:
This is awful news for my soon-to-debut review website called print.publication :cry:

Reply:
Better think of a rebrand